IDS/Config/Config.ini.php

```diff
@@ -19,8 +19,8 @@
     
     ; in case you want to use a different HTMLPurifier source, specify it here
     ; By default, those files are used that are being shipped with PHPIDS
-    HTML_Purifier_Path	= vendors/htmlpurifier/HTMLPurifier.auto.php
-    HTML_Purifier_Cache = vendors/htmlpurifier/HTMLPurifier/DefinitionCache/Serializer
+    HTML_Purifier_Path	= ../../../vendor/ezyang/htmlpurifier/library/HTMLPurifier.auto.php

```

Comment:
This one maybe problematic especially on a d8 build but not 100% sure where packages go in d8 builds